pkgs/development/libraries/zlib/default.nix

```diff
@@ -23,23 +23,17 @@ assert splitStaticOutput -> static;
 
 stdenv.mkDerivation (rec {
   pname = "zlib";
-  version = "1.2.11";
+  version = "1.2.12";
 
   src = fetchurl {
     urls =
       [ "https://www.zlib.net/fossils/zlib-${version}.tar.gz"  # stable archive path
         "mirror://sourceforge/libpng/zlib/${version}/zlib-${version}.tar.gz"
       ];
-    sha256 = "c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1";
+    sha256 = "1n9na4fq4wagw1nzsfjr6wyly960jfa94460ncbf6p1fac44i14i";
```

Comment:
Maybe it's a better idea to use the hex representation? This is the right hash, but the hex representation of a hash for a package that rarely changes is easier to verify at a glance against the zlib website (which is where I got that hash)

Reply:
Good point. Switched to base16.